### DISCUSSION TITLE
GitHub Actions log folding outer line including cache information

### DISCUSSION THREAD

**Discussion by orta** [vercel/turborepo#9173] [Ideas]
### Goals

In GitHub Actions when running a series of jobs, it (very helpfully) puts logs behind a disclosure triangle, so instead of hundreds of lines, my CI looks like:

```
• Running build in 9 packages
//:relay
nakama:build:pre
native-app:build
nakama:build
api:build:pre
api:build
studio.puzzmo.com:build
puzzmo-com:build
@puzzmo/main-runtime:build

 Tasks:    9 successful, 9 total
Cached:    1 cached, 9 total
  Time:    2m54.248s 
```

It would be nice to distinguish which of those jobs _were_ ran without having to click through into the tasks.

e.g.

```
• Running build in 9 packages
//:relay
nakama:build:pre
native-app:build
nakama:build
api:build:pre
api:build 𐄂
studio.puzzmo.com:build
puzzmo-com:build
@puzzmo/main-runtime:build

 Tasks:    9 successful, 9 total
Cached:    1 cached, 9 total
  Time:    2m54.248s 
```

### Background

-

### Proposal

However you want, probably not by me though, I don't have time to learn rust 🍡

**Comment by anthonyshew** [comment]
It sounds like you may be interested in the `--affected` flag: https://turbo.build/repo/docs/reference/run#--affected
